deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml

```diff
@@ -41,7 +41,9 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots/status"]
     verbs: ["update"]
-
+  - apiGroups: [""]
+    resources: ["nodes"]
```

Comment:
This new RBAC rule is only needed for distributed mode.  Can you comment it out and add comments on when it is needed?

Reply:
Commented it out and added comments on when to use.